apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -155,7 +155,7 @@ export class SendMessagePush extends SendMessageBase {
         continue;
       }
 
-      const overrides = command.overrides[integration.providerId] || {};
+      const overrides = Object.assign(command.overrides, command.overrides[integration.providerId]);
```

Comment:
What's the reason behind this change? I am afraid that it can led to buggy behaviour.